src/Symfony/Component/Messenger/Transport/CloseableTransportInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
```

Comment:
This should be removed as we're not using it in the Symfony code base.

Reply:
done